docs/how-it-works.mdx

```diff
@@ -0,0 +1,121 @@
+
+### Step 3: Client sends shares to the aggregators
+
+The client library sends each report share via HTTPS to the leader and helper
+aggregator.
+
+Aggregators reject out-of-range or invalid measurements, as defined in the task parameters. This is done without knowledge of the full input measurement.
+
+The aggregators in use are determined by the app developer and encoded in the
+task parameters. Each aggregator is operated independently by two different
+organizations. Often, Divvi Up will act as the leader, while another
+organization operates as a helper.
+
+<!-- TODO(#11): link BYOH guide here -->
+
+Crucially, these organizations are non-colluding. They do not directly share
+report shares or otherwise conspire to defeat the protocol. If you do not trust
+any of the already-available third-party organizations that Divvi Up partners
+with, you can self-host your own helper aggregator.
+
+:::note
+Both shares are actually sent to the leader aggregator. The leader
+aggregator relays the helper's report share to the helper. Because the helper's
+report share is encrypted to the helper, the leader cannot read it and the
+non-collusion property is preserved.
+
+This is done to simplify client library implementation and reduce the number of
+client network round-trips.
+:::
+
+### Step 4: Aggregators perform aggregation
+
+The aggregators use [multi-party computation][MPC] to work together to compute
+an aggregate across a batch of report shares, without revealing the report share
+to each other and without having any knowledge of the actual submitted
+measurement.
+
+The protocol and cryptography involved is described in [Verifiable Distributed
+Aggregation Functions (VDAF)][VDAF].
+
+Computation will yield two aggregate shares, one for each aggregator.
+
+### Step 5: Subscriber collects aggregate metrics
+
+The subscriber operates a collector, which queries each aggregator to collect
+the aggregate shares. Depending on the query type, this could be a query over a
+time interval, or for an arbitrary batch of reports.
+
+Aggregates often aren't available immediately, depending on the query type. The
+collector will poll the aggregator until it's ready.
+
+Each aggregate share is encrypted to the collector. The collector reconstitutes
+the aggregate shares, yielding the aggregate result and the number of reports
+included in the result.
+
+<!--
+TODO(https://github.com/divviup/janus-ops/issues/1005): we should make mention of the adapters we
+come up with here.
+-->
+
+Off-the-shelf collectors that more elegantly handle the collection process and
+presentation of data are currently in development.
+
+:::note
+Similar to with report shares, a collector does not directly talk to
+both aggregators. Instead the helper's aggregate share is relayed through the
+leader. Because each share is encrypted to the collector, the leader cannot read
+the aggregate shares or reconstitute the aggregate.
```

Comment:
```suggestion
As with report shares, a collector does not directly talk to
both aggregators. Instead the helper's aggregate share is relayed through the
leader. Because each share is encrypted to the collector, the leader cannot read
the aggregate shares or reconstitute the aggregate.
```